### DISCUSSION TITLE
GraphQL Examples

### DISCUSSION THREAD

**Discussion by pthrinathraj** [community/community#24585] [Apps, API and Webhooks]
<p>I’m new to GraphQL and finding it hard to write the queries and mutations without the proper examples and documentation. Let’s keep adding the examples</p>

**Comment by pthrinathraj** [comment]
<p>Any examples for graphQL  <strong>createBranchProtectionRule ??</strong></p>

**Comment by parithiban** [comment, marked as answer]
<p><a class="mention" href="/pthrinathraj">@pthrinathraj</a> You can use graphiql explorer to explore more about GitHub V4.</p>
<p>See the below example to create a branch protection rule</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/8/8eb5d44c0691393d54878756505180d5eb5bf89a.png" data-download-href="/uploads/short-url/kmtmPymKpWnixMiFy4g2bGdrOhY.png?dl=1" title="Screen Shot 2020-03-10 at 6 48 43 PM.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/7281166/181085334-4d1cb46a-dd46-4c5d-89d0-7a708959a023.png" alt="Screen Shot 2020-03-10 at 6 48 43 PM.png" data-base62-sha1="kmtmPymKpWnixMiFy4g2bGdrOhY" width="690" height="195" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/8/8eb5d44c0691393d54878756505180d5eb5bf89a_2_690x195.png, https://aws1.discourse-cdn.com/github/optimized/2X/8/8eb5d44c0691393d54878756505180d5eb5bf89a_2_1035x292.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/8/8eb5d44c0691393d54878756505180d5eb5bf89a_2_1380x390.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/8/8eb5d44c0691393d54878756505180d5eb5bf89a_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2020-03-10 at 6 48 43 PM.png</span><span class="informations">2852×810 205 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>You can check the output in the settings tab and clicking branches options in your repo.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/5/58786e3dba88f14fccc82e57b7cb4678d02a7827.png" data-download-href="/uploads/short-url/cCE2pgZX70akLamKl3snssaDEBF.png?dl=1" title="Screen Shot 2020-03-10 at 6.49.30 PM.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/7281166/181085336-b13b67f9-a668-4e5b-a3ef-93e4048c6057.png" alt="Screen Shot 2020-03-10 at 6.49.30 PM.png" data-base62-sha1="cCE2pgZX70akLamKl3snssaDEBF" width="690" height="358" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/5/58786e3dba88f14fccc82e57b7cb4678d02a7827_2_690x358.png, https://aws1.discourse-cdn.com/github/optimized/2X/5/58786e3dba88f14fccc82e57b7cb4678d02a7827_2_1035x537.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/5/58786e3dba88f14fccc82e57b7cb4678d02a7827_2_1380x716.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/5/58786e3dba88f14fccc82e57b7cb4678d02a7827_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2020-03-10 at 6.49.30 PM.png</span><span class="informations">2752×1430 363 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
